artman/conductors/cloudtasks_conductor.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Artman conductor to claim and execute remote tasks.'''
+
+import base64
```

Comment:
Please start every module with `from __future__ import absolute_import`. (I will stop repeating this on any additional modules I see, since that would be annoying, but it applies everywhere.)